src/app/pages/home/components/account-actions.tsx

```diff
@@ -43,12 +48,14 @@ export function AccountActions(props: FlexProps) {
         />
       )}
 
-      <ActionButton
-        data-testid={HomePageSelectors.SwapBtn}
-        icon={<SwapIcon />}
-        label="Swap"
-        onClick={() => navigate(RouteUrls.Swap)}
-      />
+      {isMainnet && (
+        <ActionButton
+          data-testid={HomePageSelectors.SwapBtn}
+          icon={<SwapIcon />}
+          label="Swap"
+          onClick={() => navigate(RouteUrls.Swap)}
+        />
+      )}
```

Comment:
We have a `whenNetwork` helper you can use here to avoid the equality expression on L26